README.md

```diff
@@ -287,12 +286,13 @@ Releasing a new version is quite simple, as the packages are build and distribut
 * minor release (second version part increases): 1.1.3 -> 1.2.0
 * major release (first version part increases): 1.2.0 -> 2.0.0
 
-*Release steps*:
+### Cutting a Major or Minor release
+
 1. Make sure all dependencies are up to date (**especially Rasa SDK**)
     - For Rasa SDK that means first creating a [new Rasa SDK release](https://github.com/RasaHQ/rasa-sdk#steps-to-release-a-new-version) (make sure the version numbers between the new Rasa and Rasa SDK releases match)
     - Once the tag with the new Rasa SDK release is pushed and the package appears on [pypi](https://pypi.org/project/rasa-sdk/), the dependency in the rasa repository can be resolved (see below).
 2. Switch to the branch you want to cut the release from (`master` in case of a major / minor, the current feature branch for micro releases) 
-    - Update the `rasa-sdk` entry in `pyproject.toml` with the new release version and run `poetry update`. This creates a new `poetry.lock` file with all dependencies resolved.
+    - If there is an updated `rasa-sdk` dependency, update the `rasa-sdk` entry in `pyproject.toml` with the new release version and run `poetry update`. This creates a new `poetry.lock` file with all dependencies resolved.
```

Comment:
I'm not sure this one is accurate, as far as I know we aim to keep version parity between `rasa` and `rasa-sdk` 🤔